### DISCUSSION TITLE
minha senha de adm não pode ser mudada, clico em nova senha e nada acontece, a senha não muda

### DISCUSSION THREAD

**Discussion by Rardotz** [RamonSilva20/mapos#2194] [Q&A]
quando eu abro o mapos e tento fazer uma nova senha do adm eu não consigo. Eu vou em mudar a senha preencho tudo oque tem para preencher e até da o alerta de senha mudada com sucesso, porém a senha nova não funciona, eu faço as tentativas, mas não da certo e só consigo logar com a senha antiga, mesmo tendo tentado mudar ela varias vezes, eu já tentei desinstalar e instalar de novo e nada, já subi no xamp(localmente), laragon(localmente) e até na 000webhost(servidor-web) mas todas estão com o mesmo problema. detalhe que não da erro em nenhum momento. até tentei excluir e apagar manualmente o usuário adm para criar outro em seguida na intenção de que a nova senha funcionasse nesse usuário adm mas nada também

**Comment by Rardotz** [comment]
detalhe que já instalei o mapos em vários computadores para testar e todos apresentaram esse problema

**Comment by NagatoInformatica** [comment]
Rardotz tenho o mesmo problema
